clients/venice-thin-client/src/main/java/com/linkedin/venice/client/store/StatTrackingStoreClient.java

```diff
@@ -225,30 +245,32 @@ public void compute(
         preRequestTimeInNS);
   }
 
+  private static void handleUnhealthyRequest(
+      ClientStats clientStats,
+      Throwable throwable,
+      double latency,
+      int keyCount) {
+    int httpStatus = getUnhealthyRequestHttpStatus(throwable);
+    clientStats.emitUnhealthyRequestMetrics(latency, keyCount, httpStatus);
+    if (throwable instanceof VeniceClientHttpException) {
+      clientStats.recordHttpRequest(httpStatus);
```

Comment:
It seems both `emitUnhealthyRequestMetrics`  and `recordHttpRequest` would emit tehuti metrics based on `http status`, so shall we get rid of the latter?

Reply:
Good call out. I will check on this.

Reply:
Will thin client emit duplicate metrics when otel is enabled?

Reply:
thin client will emit both tehuti and otel metric (if otel is enabled) until we go ahead and remove tehuti. I will add a feature flag to disable tehuti metrics registration so we don't have to dual emit until we end up removing tehuti. I will cut a separate PR for that.

Reply:
SG, thanks